app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
```

Comment:
Consider providing a custom error message for `NotVaccinatedError` to make it more informative when raised.